.eslintrc.cjs

```diff
@@ -0,0 +1,128 @@
+/** @type {import("@types/eslint").Linter.Config} */
+module.exports = {
```

Comment:
I moved this out of `package.json` and into its own file so that we could use things like single quotes and comments.

Reply:
I don't mind about this. The file is getting very big and we can take advantage of single quotes and comments.
What do you think @sergioramos?

Reply:
The reason I suggested in `package.json` was to not have tons of root files. But I'm OK either way.